config.js

```diff
@@ -30,11 +30,11 @@ var config = {
     server: {
```

Comment:
CNR entry regarding the changes in this PR is missing.

Reply:
I understand mentioning not only the removal of the legacy expression language, but also the fix of the bidirectional plugin to work with JEXL.

Reply:
I understand, it should mention issue #1340